Fix project gitlab ubuntu CI since PyGObject >= 3.52.1 now depends on libgirepository-2.0-dev

On current latest Ubuntu, for python 3.8, pip brings PyGObject below 3.52.1 (ie PyGObject 3.48.2) which depends on libgirepository-1.0-dev while for python 3.13, PyGObject is above 3.52.1 (currently 3.52.3) which depends on libgirepository-2.0-dev (the latter has removed its dependency on gobject-introspection so it has to be explicitly added too).

python 3.13 job error when libgirepository-2.0-dev is not installed instead of hte version 1.0:
```
Run make install_pip PIP_ARGS='.["test"]'
Installing Solaar via pip
python -m pip install --upgrade pip
Requirement already satisfied: pip in /opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages (25.0.1)
pip install .["test"]
Processing /home/runner/work/Solaar/Solaar
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Collecting evdev>=1.1.2 (from solaar==1.1.14)
  Downloading evdev-1.9.1.tar.gz (33 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Collecting pyudev>=0.13 (from solaar==1.1.14)
  Downloading pyudev-0.24.3-py3-none-any.whl.metadata (4.6 kB)
Collecting PyYAML>=3.12 (from solaar==1.1.14)
  Downloading PyYAML-6.0.2-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.1 kB)
Collecting python-xlib>=0.27 (from solaar==1.1.14)
  Downloading python_xlib-0.33-py2.py3-none-any.whl.metadata (6.2 kB)
Collecting psutil>=5.4.3 (from solaar==1.1.14)
  Downloading psutil-7.0.0-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (22 kB)
Collecting dbus-python (from solaar==1.1.14)
  Downloading dbus-python-1.4.0.tar.gz (232 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Collecting PyGObject (from solaar==1.1.14)
  Downloading pygobject-3.52.3.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 34.2 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'error'
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [222 lines of output]
      + meson setup /tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556 /tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556/.mesonpy-jt457wk8 -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md -Dtests=false -Dwheel=true --wrap-mode=nofallback --native-file=/tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556/.mesonpy-jt457wk8/meson-python-native-file.ini
      The Meson build system
      Version: 1.7.2
      Source dir: /tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556
      Build dir: /tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556/.mesonpy-jt457wk8
      Build type: native build
      Project name: pygobject
      Project version: 3.52.3
      C compiler for the host machine: cc (gcc 13.3.0 "cc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0")
      C linker for the host machine: cc ld.bfd 2.42
      Host machine cpu family: x86_64
      Host machine cpu: x86_64
      Program python3 found: YES (/opt/hostedtoolcache/Python/3.13.2/x64/bin/python)
      Found pkg-config: YES (/usr/bin/pkg-config) 1.8.1
      Run-time dependency python found: YES 3.13
      Found CMake: /usr/local/bin/cmake (3.31.6)
      Run-time dependency girepository-2.0 found: NO (tried pkgconfig and cmake)
      Not looking for a fallback subproject for the dependency girepository-2.0 because:
      Use of fallback dependencies is disabled.
      
      ../meson.build:31:9: ERROR: Dependency 'girepository-2.0' is required but not found.
      
      A full log can be found at /tmp/pip-install-xc2wxon8/pygobject_b3773d22a8d64176930fc1c040520556/.mesonpy-jt457wk8/meson-logs/meson-log.txt
==== CI platform detected, click here for meson-log.txt contents. ====
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
make: *** [Makefile:34: install_pip] Error 1
Error: Process completed with exit code 2.
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -19,9 +19,15 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install Ubuntu dependencies
+    - name: Install Ubuntu dependencies for python 3.8
+      if: matrix.python-version == '3.8'
       run: |
         make install_apt
+
+    - name: Install Ubuntu dependencies for python 3.13
+      if: matrix.python-version == '3.13'
+      run: |
+        make install_apt_python3.13
 
     - name: Install Python dependencies
       run: |

--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,12 @@ install_macos: install_brew install_pip
 install_apt:
 	@echo "Installing Solaar dependencies via apt"
 	sudo apt update
-	sudo apt install libdbus-1-dev libglib2.0-dev libgtk-3-dev libgirepository1.0-dev
+	sudo apt install libdbus-1-dev libglib2.0-dev libgtk-3-dev libgirepository-1.0-dev
+
+install_apt_python3.13:
+	@echo "Installing Solaar dependencies via apt"
+	sudo apt update
+	sudo apt install libdbus-1-dev libglib2.0-dev libgtk-3-dev libgirepository-2.0-dev gobject-introspection
 
 install_dnf:
 	@echo "Installing Solaar dependencies via dn"

